site/src/_includes/components/over-prefetching.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "layouts/normal-section-wrapper.njk" %}
+{% block section %}
+{% sectionTitle "Concerned about over-prefetching? We've got you covered" %}
+{{ "By default `quicklink` observes all in-viewport links in `document.body`. There are different ways of telling `quicklink` to limit the number of links to prefetch." | markdown | safe }}
+
+{{ "The most common approach is passing passing different `options` to configure prefetching when calling `quicklink.listen()`:" | markdown | safe }}
+
+{{ "* Indicating a specific DOM element to observe, with the `options.el` parameter:" | markdown | safe }}
+
+{% markdownConvert %}
+```js
+quicklink.listen({
+  el: document.getElementById('carousel')
```

Comment:
Wdyt about us using an example like `document.getElementById('article')` or `document.getElementById('content')`?

Reply:
Changed it to 'content', thank you!